Week-4/Javascript_basics/maps/script.js

```diff
@@ -0,0 +1,21 @@
+const calcWordFrequencies = () => {
+  let question = prompt('Enter the input');
+  if (question) {
+    const questionArray = question.split(' ');
+    var freq = {};
+    questionArray.map(word => {
+      if (freq[word]) {
+        freq[word] += 1;
+      } else {
+        freq[word] = 1;
+      }
+    });
+    let OutputArr = Object.entries(freq);
+    for ([key, value] of OutputArr) {
+      console.log(key, value);
+    }
+    return 'Frequencies returned';
+  } else {
+    return 'No input';
+  }
+};
```

Comment:
function call missing

Reply:
Function call i am performing in the html console

Reply:
okay, but the new person might not know that, lets invoke the function in script and in html you can inform what is getting logged.